src/osd/OSD.cc

```diff
@@ -1808,73 +1813,74 @@ void OSDService::queue_for_rep_scrub_resched(PG* pg,
 					     Scrub::act_token_t act_token)
 {
   // Resulting scrub event: 'SchedReplica'
+  uint64_t cost = cct->_conf->osd_scrub_event_cost;
   queue_scrub_event_msg<PGRepScrubResched>(pg, with_priority, qu_priority,
```

Comment:
Why isn't this a queue_simple_scrub_event_msg?

Also, please move the rename from queue_scrub_event_msg->queue_simple_scrub_event_msg to a separate commit preceding your main change.  In general, *whenever* you can move a self-contained refactor to its own commit please do so especially if it touches a bunch of lines of code irrelevant to the rest of the PR.

Also, I think I probably suggested queue_simple_scrub_event_msg, but let's instead call it queue_scrub_event_msg_default_cost.